pg_analytics/tests/basic.rs

```diff
@@ -378,3 +377,104 @@ fn vacuum(mut conn: PgConnection) {
     "DROP TABLE t, s".execute(&mut conn);
     "VACUUM".execute(&mut conn);
 }
+
+#[rstest]
+#[ignore]
+async fn copy_out_arrays(mut conn: PgConnection) {
+    ResearchProjectArraysTable::setup().execute(&mut conn);
+
+    let copied_csv = conn
+        .copy_out_raw(
+            "COPY (SELECT * FROM research_project_arrays) TO STDOUT WITH (FORMAT CSV, HEADER)",
+        )
+        .await
+        .unwrap()
+        .to_csv();
+
+    let expected_csv = "\
+experiment_flags,notes,keywords,short_descriptions,participant_ages,participant_ids,observation_counts,measurement_errors,precise_measurements
+\"{f,t,f}\",\"{\"\"Need to re-evaluate methodology\"\",\"\"Unexpected results in phase 2\"\"}\",\"{\"\"sustainable farming\"\",\"\"soil health\"\"}\",\"{\"\"FARMEX    \"\",\"\"SOILQ2    \"\"}\",\"{22,27,32}\",\"{201,202,203}\",\"{160,140,135}\",\"{0.025,0.02,0.01}\",\"{2,2.1,2.2}\"
+\"{t,f,t}\",\"{\"\"Initial setup complete\"\",\"\"Preliminary results promising\"\"}\",\"{\"\"climate change\"\",\"\"coral reefs\"\"}\",\"{\"\"CRLRST    \"\",\"\"OCEAN1    \"\"}\",\"{28,34,29}\",\"{101,102,103}\",\"{150,120,130}\",\"{0.02,0.03,0.015}\",\"{1.5,1.6,1.7}\"";
+
+    assert_eq!(copied_csv.trim(), expected_csv);
+}
```

Comment:
You should use a [raw string literal](https://rahul-thakoor.github.io/rust-raw-string-literals/) here to avoid escaping quotes.

Reply:
Ooh thank you!